renku/data/shacl_shape.json

```diff
@@ -787,6 +787,54 @@
                "minCount": 1
             }
          ]
+      },
+      {
+         "@id": "_:invalidationShape",
+         "@type": "sh:NodeShape",
+         "ignoredProperties": [
+            {
+               "@id": "rdf:type"
+            }
+         ],
+         "closed": true,
+         "targetClass": "prov:Invalidation",
+         "property": [
+            {
+               "path": {
+                  "sh:inversePath": {
+                     "@id": "prov:qualifiedInvalidation"
+                  }
+               },
+               "nodeKind": "sh:BlankNodeOrIRI"
+            },
+            {
+               "sh:class": {
+                  "@id": "prov:Activity"
+               },
+               "path": "prov:activity",
+               "minCount": 1
+            }
+         ]
+      },
+      {
+         "@id": "_:entityShape",
+         "@type": "sh:NodeShape",
+         "ignoredProperties": [
+            {
+               "@id": "rdf:type"
+            }
+         ],
+         "closed": false,
+         "targetClass": "prov:Entity",
```

Comment:
I guess this is just the beginning of the `Entity` shape?

Reply:
Since so many things can be entities, I didn't want to be restrictive. We can add properties as needed, though the Entity class itself doesn't have any jsonld properties on its own.